app/_src/kubernetes-ingress-controller/guides/requests/rewrite-host.md

```diff
@@ -105,14 +122,58 @@ You can set the Host header explicitly if needed by disabling `konghq.com/preser
 
 There are three options to rewrite the default path handling behavior:
 
+{% if_version gte 3.2.0 %}
+* Rewrite using Gateway API's `URLRewrite` filter 
+{% endif_version %}
 * Rewrite using regular expressions
 * Remove the path prefix using `strip-path`
 * Add a path prefix using the `path` annotation
 
+{% if_version gte 3.2.0 %}
+### Rewrite using Gateway API's `URLRewrite` filter
+
+#### Rewriting full path
+
+Add the `URLRewrite` filter with `path.replaceFullPath` to your `HTTPRoute` rule to rewrite path entirely.
+
+```yaml
+...
+filters:
+- type: URLRewrite
+  urlRewrite:
+    path:
+      type: ReplaceFullPath
+      replaceFullPath: /rewritten-path
+```
+
+#### Rewriting path prefix
+
+Add the `URLRewrite` filter with `path.replacePrefixMatch` to your `HTTPRoute` rule to rewrite path prefix.
+See the [URLRewrite filter documentation](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.HTTPPathModifier)
+for more information.
+
+```yaml
+...
+rules:
+  - matches:
+      - path:
+          type: PathPrefix # Only PathPrefix path type is supported with URLRewrite filter using path.type == ReplacePrefixMatch.
+          value: /old-prefix
+    filters:
+     - type: URLRewrite
+       urlRewrite:
+         path:
+           type: ReplacePrefixMatch
+           replacePrefixMatch: /new-prefix
+```
+
+{% endif_version %}
+
 ### Rewrite using regular expressions
 
 {:.note}
 > This feature is available from {{ site.kic_product_name }} 2.12 and requires the [`RewriteURIs` feature gate](/kubernetes-ingress-controller/{{ page.release }}/reference/feature-gates/) to be activated.
+> It's only available for Ingress.
```

Comment:
This is misleading: line 183 contains the example with HTTPRoute. Do we support `konghq.com/rewrite` for `HTTPRoute`s?

Reply:
Good catch. `HTTPRoute` example should be removed as we do not support `konghq.com/rewrite` as stated in the note.